appcli/orchestrators.py

```diff
@@ -43,7 +44,7 @@ def start(self, cli_context: CliContext, container: str = ()) -> CompletedProces
         Starts Docker containers.
 
         Args:
-            cli_context (CliContext): Context for this CLI run.
+            cli_context (CliContext): The current cli context.
```

Comment:
Use uppercase `CLI` in your comments for the acronym

Reply:
Fixed all over appcli now